kuzzleio/kuzzle-device-manager
# feat: digitaltwin exports

## What does this PR do ?

Add the list exports for digitalTwins

### How should this be manually tested?

  - Step 1 : Call in `POST` the endpoint `/device-manager/<engineId>/<type>/_export`, with `query` and `sort` in body
  - Step 2 : Call the URL returned by previous request, you should get a CSV file

`<type>` is `assets` or `devices`

### Other changes

Refactor `MeasureExporter` with `AbstractExporter` to mutualize the exports logic
